kafka/test_kafka.py

```diff
@@ -9,7 +10,15 @@
 class KafkaTestCase(DataprocTestCase):
     COMPONENT = 'kafka'
     INIT_ACTIONS = ['kafka/kafka.sh']
+    KAFKA_CRUISE_CONTROL_INIT_ACTION = ['kafka/kafka.sh', 'kafka/cruise-control.sh']
+    KAFKA_MANAGER_INIT_ACTION = ['kafka/kafka.sh', 'kafka/kafka-manager.sh']
     TEST_SCRIPT_FILE_NAME = 'validate.sh'
+    KAFKA_PYTHON_INIT_ACTION_PATH = 'gs://dataproc-kafka-test/kafka_python.sh'
```

Comment:
I think we should merge the 2 init actions into one, pass parameters or detect the env inside the init action.

Reply:
merging them would require to change the script at the location at which it is stored as it is custom made init action script for kafka and while running with the repo it is not able to fetch this file. So,

Reply:
I don't quite understand why it needs to be in this GCS bucket. Cannot it simply use the one from this repo? Just like the kafka.sh itself.

Reply:
I tried that but it started throwing error saying file not found.

Reply:
It should be the same as the kafka init action itself. Or you can extend the kafka init action with parameters to allow you do the work, this will bypass the problem and mabye some users also like it.

Reply:
Passing the parameter and based on the dataproc image version being used for the init action, installing the kafka-python package accordingly with kafka init action itself.